python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -486,6 +619,97 @@ def run(self, fakeCat, image, wcs, photoCalib):
 
         return resultStruct
 
+    def _standardizeColumns(self, fakeCat, band):
+        """Use config variables to 'standardize' the expected columns and column
+        names in the input catalog.
+
+        Parameters
+        ----------
+        fakeCat : `pandas.core.frame.DataFrame`
+            The catalog of fake sources to be input
+        band : `str`
+            Label for the current band being processed.
+
+        Returns
+        -------
+        outCat : `pandas.core.frame.DataFrame`
+            The standardized catalog of fake sources
+        """
+        cfg = self.config
+        replace_dict = {}
+
+        # Prefer new config variables over deprecated config variables.
+        # The following are fairly simple to handle as they're just column name
+        # changes.
+        for new_name, depr_name, std_name in [
+            (cfg.ra_col, cfg.raColName, 'ra'),
+            (cfg.dec_col, cfg.decColName, 'dec'),
+            (cfg.bulge_n_col, cfg.nBulge, 'bulge_n'),
+            (cfg.bulge_pa_col, cfg.paBulge, 'bulge_pa'),
+            (cfg.disk_n_col, cfg.nDisk, 'disk_n'),
+            (cfg.disk_pa_col, cfg.paDisk, 'disk_pa'),
+            (cfg.mag_col%band, cfg.magVar%band, 'mag'),
+            (cfg.select_col, cfg.sourceSelectionColName, 'select')
+        ]:
+            # Only standardize "select" column if doSubSelectSources is True
+            if not cfg.doSubSelectSources and std_name == 'select':
+                continue
+            if new_name in fakeCat.columns:
+                replace_dict[new_name] = std_name
+            elif depr_name in fakeCat.columns:
+                replace_dict[depr_name] = std_name
+            else:
+                raise ValueError(f"Could not determine column for {std_name}.")
+        fakeCat = fakeCat.rename(columns=replace_dict, copy=False)
+
+        # Handling the half-light radius and axis-ratio are trickier, since we
+        # moved from expecting (HLR, a, b) to expecting (semimajor, axis_ratio).
+        # Just handle these manually.
+        if (
+            cfg.bulge_semimajor_col in fakeCat.columns
+            and cfg.bulge_axis_ratio_col in fakeCat.columns
+        ):
+            fakeCat = fakeCat.rename(
+                columns={
+                    cfg.bulge_semimajor_col: 'bulge_semimajor',
+                    cfg.bulge_axis_ratio_col: 'bulge_axis_ratio',
+                    cfg.disk_semimajor_col: 'disk_semimajor',
+                    cfg.disk_axis_ratio_col: 'disk_axis_ratio',
+                },
+                copy=False
+            )
+        elif (
+            cfg.bulgeHLR in fakeCat.columns
+            and cfg.aBulge in fakeCat.columns
+            and cfg.bBulge in fakeCat.columns
+        ):
+            fakeCat['bulge_axis_ratio'] = \
```

Comment:
https://developer.lsst.io/python/style.html#python-s-implied-continuation-inside-parens-brackets-and-braces-should-be-used-for-wrapped-lines, backslashes for line continuation are not preferred.